bin/docker-entrypoint

```diff
@@ -4,9 +4,10 @@ set -e
 worker() {
   WORKERS_COUNT=${WORKERS_COUNT:-2}
   QUEUES=${QUEUES:-queries,scheduled_queries,celery,schemas}
+  MAX_MEMORY=$(($(/usr/bin/awk '/MemTotal/ {print $2}' /proc/meminfo)/4))
 
   echo "Starting $WORKERS_COUNT workers for queues: $QUEUES..."
-  exec /usr/local/bin/celery worker --app=redash.worker -c$WORKERS_COUNT -Q$QUEUES -linfo --maxtasksperchild=10 -Ofair
+  exec /usr/local/bin/celery worker --app=redash.worker -c$WORKERS_COUNT -Q$QUEUES -linfo --max-tasks-per-child=10 --max-memory-per-child=$MAX_MEMORY -Ofair
```

Comment:
So `--maxtasksperchild` was simply ignored or they had some backward compatibility support in place?

Reply:
I don't know, I'm just porting a change @jasonthomas has applied to our fork.

Reply:
There is backward compatibility in place as per [1].

[1] https://github.com/celery/celery/blob/8047ca9e818db8151dc6c80814600797ab94f5bc/docs/history/whatsnew-4.0.rst#worker

Reply:
Thanks @jasonthomas!